app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe):
+    masks_to_buy = sum([
+        1
+        for friend in friends
+        if not friend["wearing_a_mask"] or "wearing_a_mask" not in friend
```

Comment:
There you check if visitor wear mask twice. First time in go_to_cafe function, second time inside visit cafe function